Updated script

michelaparma opened the pull request:
Problems solved:
1. Missing headers and footers
2. Links to digitised images
3. Wrong symbol ⁓ replaced by the correct one
4. Two chapter titles on the same line

michelaparma committed:
Updated script

Problems solved:
1. Missing headers and footers
2. Links to digitised images
3. Wrong symbol ⁓ replaced by the correct one
4. Two chapter titles on the same line

michaelscho reviewed (changes requested):
Regex needs changes and templates should not be deleted

michelaparma committed:
Updated script

Problems solved:
1. Missing headers and footers
2. Links to digitised images
3. Wrong symbol ⁓ replaced by the correct one
4. Two chapter titles on the same line
